extensions/wikia/AdEngine/js/wad/babDetection.js

```diff
@@ -1,13 +1,13 @@
 /*global define*/
-define('ext.wikia.adEngine.babDetection', [
+define('ext.wikia.adEngine.wad.babDetection', [
```

Comment:
It isn't safe for mobile-wiki

Reply:
You right. I prepared additional PRs for mobile-wiki.